admin/python/lsst/qserv/admin/qservAdmin.py

```diff
@@ -486,7 +494,9 @@ def _createTable(self, options, dbName, tableName):
     def getNode(self, nodeName):
         """
         Returns node definition for specified node name. Returned dictionary has
-        keys defined in https://dev.lsstcorp.org/trac/wiki/db/Qserv/CSS#Node-related
+        keys defined in https://dev.lsstcorp.org/trac/wiki/db/Qserv/CSS#Node-related.
+        The state of the node (which does not have separate key in CSS) is returned
+        as 'state' key in the dictionary.
```

Comment:
I don't think that sphinx is the right place for that, for me sphinx works better only for documentation that is release-specific. Editing sphinx docs and distributing them is much harder that wiki. Can you easily fix an error in sphinx documentation in the last 5 releases that we built?